docs/user-guide/annotations.md

```diff
@@ -120,26 +119,28 @@ Click on <img src="../img/button/apply_button.jpg" class="ms-docbutton"/> to app
 Managing Annotations
 --------------------
 
-You can manage anytime your annotations through the *Annotation Viewer*. Click on some annotation to display it.
+Once annotations are added to the [TOC](toc.md), the user can **Manage** it by clicking to <img src="../img/button/edit_button.jpg" class="ms-docbutton" /> button from the TOC toolbar and the *Main Annotations panel* will be open.
 
-<img src="../img/annotations/annotation_toolbar.jpg" class="ms-docimage" style="max-width:500px;"/>
+<img src="../img/annotations/annotations_main_panel.jpg" class="ms-docimage" style="max-width:500px;" />
 
-* You can edit your annotations by clicking the geometry card from list of geometries of the annotation. It allows you to change the geometry and/or the information related to the annotations. See the gif below to better understand how:
+From it, the editor is allowed to:
 
-<img src="../img/annotations/editing_annotations.gif" class="ms-docimage" style="max-width:700px;" />
+* **Download** a file with all the existing annotations by clicking on <img src="../img/button/download_annotation_button.jpg" class="ms-docbutton" > button
+
+* **Upload** annotations from a valid `json` file by clicking on <img src="../img/button/upload_annotation_button.jpg" class="ms-docbutton" > button
 
-* The <img src="../img/button/zoom_button.jpg" class="ms-docbutton" /> button allow the map to zoom on the annotation.
+* **Zoom** an annotation on map by clicking on <img src="../img/button/white-zoom.jpg" class="ms-docbutton" /> button
 
-* The <img src="../img/button/eyeon.jpg" class="ms-docbutton" /> button allow the user to show/hide the annotation on the map.
+* **Show/Hide** an annotation on the map by clicking on <img src="../img/button/eyeon.jpg" class="ms-docbutton" /> button
 
-* Annotations can be also download in `json` format and reused in other maps. Using <img src="../img/button/download_annotation_button.jpg" class="ms-docbutton" /> button you will be able to export your annotation in the `Annotation.json` file and download it.
+From the *Main Annotations Panel*, by selecting an annotation from the list, the user is returned to the *Annotation Viewer* where the annotation can be edited.
 
-<img src="../img/annotations/download_annotation.jpg" class="ms-docimage"/>
+<img src="../img/annotations/annotation_toolbar.jpg" class="ms-docimage" style="max-width:500px;"/>
 
-From the main panel of the *Annotations* tool you can also download/upload annotations to/from a `json` file.
+In particular, the editor can:
```

Comment:
isn't this part a bit mixed to Styling annotations, maybe you can move the gif here to that section